src/ARCtrl/Templates/Templates.Json.fs

```diff
@@ -0,0 +1,95 @@
+﻿module ARCtrl.Templates.Json
+
+open ARCtrl.Templates
+open ARCtrl.ISA
+open System
+#if FABLE_COMPILER
+open Thoth.Json
+#else
+open Thoth.Json.Net
+#endif
+
+module Organisation =
+    let encode = Encode.Auto.generateEncoder<Organisation>()
+
+    let decode = Decode.Auto.generateDecoder<Organisation>()
+
+module CompositeCell =
+
+    let encode = Encode.Auto.generateEncoder<CompositeCell>()
+
+    let decode = Decode.Auto.generateDecoder<CompositeCell>()
+
+module CompositeHeader =
+
+    let encode = Encode.Auto.generateEncoder<CompositeHeader>()
+
+    let decode = Decode.Auto.generateDecoder<CompositeHeader>()
+
+module ArcTable =
+
+    let encode (table: ArcTable) =
+        let keyEncoder : Encoder<int*int> = Encode.tuple2 Encode.int Encode.int
+        let valueEncoder = CompositeCell.encode
+        Encode.object [
+            "name", Encode.string table.Name
+            "header", Encode.list [
+                for h in table.Headers do yield CompositeHeader.encode h
+            ]
+            "values", Encode.map keyEncoder valueEncoder ([for KeyValue(k,v) in table.Values do yield k, v] |> Map)
+        ] 
+
+    let decode : Decoder<ArcTable> =
+        Decode.object(fun get ->
+            let decodedHeader = get.Required.Field "header" (Decode.list CompositeHeader.decode) |> ResizeArray
+            let keyDecoder : Decoder<int*int> = Decode.tuple2 Decode.int Decode.int
+            let valueDecoder = CompositeCell.decode
+            let decodedValues = get.Required.Field "values" (Decode.map' keyDecoder valueDecoder) |> System.Collections.Generic.Dictionary
+            ArcTable.create(
+                get.Required.Field "name" Decode.string,
+                decodedHeader,
+                decodedValues
+            )
+        )
+
+module Json =
```

Comment:
Why this break in module naming?

Reply:
Ups, thats a leftover from my testfile. Fixed!